app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TournamentsController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+
+  def index
+  end
+
+  def create
+    @tournament = Tournament.new(company_name: params[:company_name], start_date: params[:start_date], end_date: params[:end_date])
```

Comment:
You don't need @tournament
Also you can use `create` instead, which is like a combination of `new` and `save`
Also prefer `create!` to `create`, which raises an error so if there are validation errors they don't fail silently
```suggestion
    Tournament.create!(company_name: params[:company_name], start_date: params[:start_date], end_date: params[:end_date])
```